ops/pebble.py

```diff
@@ -752,5 +872,214 @@ def get_services(self, names: List[str] = None) -> List[ServiceInfo]:
         query = None
         if names is not None:
             query = {'names': ','.join(names)}
-        result = self._request('GET', '/v1/services', query)
-        return [ServiceInfo.from_dict(info) for info in result['result']]
+        resp = self._request('GET', '/v1/services', query)
+        return [ServiceInfo.from_dict(info) for info in resp['result']]
+
+    def read_content(self, path: str, encoding: str = 'utf-8') -> typing.Union[bytes, str]:
```

Comment:
In previous conversations we tried to keep the API here consistent with the command line (the pebble command) and the API itself when that was possible. This sounds like a case we can use that idea again: we spent a while discussing the details of the method names, CLIs (push, pull, etc), so we might as well reuse these insights to produce an API here that will resemble what we have all around. It will make the muscle memory stronger.

Reply:
See below.